chapters/shaders.adoc

```diff
@@ -2498,6 +2498,86 @@ In other shader stages, each invocation in a subgroup must: be in the same
 
 Only <<limits-subgroup-supportedStages, shader stages that support subgroup
 operations>> have defined subgroups.
+
+[NOTE]
+.Note
+====
+In shaders, there are two kinds of uniformity that is of interest to
+applications, subgroup uniform and dynamically uniform.
+
+While dynamically uniform appears to be a stronger kind of uniform
+than subgroups, it is not necessarily the case for graphics pipelines.
```

Comment:
We have graphics stages with workgroups (TS, MS, tessellation stuff), and non-graphics stages without workgroups (RT)

```suggestion
While dynamically uniform appears to be a stronger kind of uniform
than subgroups, it is not necessarily the case for shader stages without defined workgroups.
```

Reply:
"stronger" seems poorly defined in this context - is there a more specific meaning? It sounds like "better".

Reply:
> "stronger" seems poorly defined in this context - is there a more specific meaning? It sounds like "better".

"Stronger" here appears to mean "implies", so perhaps something along the lines of "While dynamically uniform appears to imply subgroup uniform" would be better?

Reply:
I was still hoping to improve the "stronger" term - something like @nanokatze suggests would be better.